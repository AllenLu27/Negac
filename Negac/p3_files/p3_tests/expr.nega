int main(int argc, bool argv) {
    x = y * z - t / r * -y;
}
