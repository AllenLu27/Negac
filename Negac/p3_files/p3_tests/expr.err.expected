FATAL [2,5]: Undeclared identifier
FATAL [2,9]: Undeclared identifier
FATAL [2,13]: Undeclared identifier
FATAL [2,17]: Undeclared identifier
FATAL [2,21]: Undeclared identifier
FATAL [2,26]: Undeclared identifier
Semantic analysis failed
